pocs/utils/rs232.py

```diff
@@ -30,7 +30,7 @@ def _parse_json(line, logger, min_error_pos=0):
 
 # Note: get_serial_port_info is replaced by tests to override the normal
 # behavior, so don't change the name without fixing the tests.
-def get_serial_port_info():
+def get_serial_port_info():  # pragma: no cover
```

Comment:
We could exercise it in a test, even if it returns an empty list. What do you think?